cabal.project

```diff
@@ -81,52 +81,55 @@ constraints:
-  , cardano-binary-test == 1.3.0
-  , cardano-crypto-class  == 2.0.0.1
-  , cardano-crypto-praos  == 2.0.0.0.1
-  , cardano-crypto-tests  == 2.0.0.0.1
-  , cardano-slotting  == 0.1.0.0
-  , measures == 0.1.0.0
-  , orphans-deriving-via == 0.1.0.0
-  , strict-containers == 0.1.0.0
-  , plutus-core == 1.0.0.1
-  , plutus-ledger-api == 1.0.0.1
-  , plutus-tx == 1.0.0.0
-  , plutus-tx-plugin == 1.0.0.0
-  , prettyprinter-configurable == 0.1.0.0
-  , plutus-ghc-stub == 8.6.5
-  , word-array == 0.1.0.0
-  , word-array == 0.1.0.0
+  , cardano-prelude >= 0.1.0.1
+  , base-deriving-via >= 0.1.0.0
+  , cardano-binary >= 1.5.0
+  , cardano-binary-test >= 1.3.0
+  , cardano-crypto-class >= 2.0.0.1
+  , cardano-crypto-praos >= 2.0.0.0.1
+  , cardano-crypto-tests >= 2.0.0.0.1
+  , cardano-slotting >= 0.1.0.0
+  , measures >= 0.1.0.0
+  , orphans-deriving-via >= 0.1.0.0
+  , strict-containers >= 0.1.0.0
+  , plutus-core >= 1.0.0.1
+  , plutus-ledger-api >= 1.0.0.1
+  , plutus-tx >= 1.0.0.0
+  , plutus-tx-plugin >= 1.0.0.0
+  , prettyprinter-configurable >= 0.1.0.0
+  , plutus-ghc-stub >= 8.6.5
+  , word-array >= 0.1.0.0
+  , word-array >= 0.1.0.0
   , typed-protocols >= 0.1.0.1
   , typed-protocols-examples >= 0.1.0.1
 
-  , cardano-ledger-alonzo == 0.1.1.1
-  , cardano-ledger-alonzo-test == 0.1.1.1
-  , cardano-ledger-babbage == 0.1.1.1
-  , cardano-ledger-babbage-test == 0.1.1.1
-  , byron-spec-chain == 0.1.1.1
-  , cardano-crypto-wrapper == 1.4.1
-  , cardano-crypto-test == 1.4.1
-  , byron-spec-ledger == 0.1.1.1
-  , cardano-ledger-byron == 0.1.1.1
-  , cardano-ledger-byron-test == 1.4.1
-  , cardano-ledger-conway == 0.1.1.1
-  , cardano-ledger-shelley-ma == 0.1.1.1
-  , cardano-ledger-shelley-ma-test == 0.1.1.1
-  , cardano-ledger-shelley == 0.1.1.1
-  , cardano-ledger-shelley-test == 0.1.1.1
-  , cardano-data == 0.1.1.1
-  , cardano-ledger-core == 0.1.1.1
-  , cardano-ledger-pretty == 0.1.1.1
-  , cardano-protocol-tpraos == 0.1.1.1
-  , non-integral == 0.1.1.1
-  , set-algebra == 0.1.1.1
-  , small-steps == 0.1.1.1
-  , small-steps-test == 0.1.1.1
-  , vector-map == 0.1.1.1
+  , cardano-ledger-alonzo >= 0.1.1.1
+  , cardano-ledger-alonzo-test >= 0.1.1.1
+  , cardano-ledger-babbage >= 0.1.1.1
+  , cardano-ledger-babbage-test >= 0.1.1.1
+  , byron-spec-chain >= 0.1.1.2
+  , cardano-crypto-wrapper >= 1.4.1
+  , cardano-crypto-test >= 1.4.2
+  , byron-spec-ledger >= 0.1.1.2
+  , cardano-ledger-byron >= 0.1.1.1
+  , cardano-ledger-byron-test >= 1.4.1
+  , cardano-ledger-conway >= 0.1.1.1
+  , cardano-ledger-shelley-ma >= 0.1.1.1
+  , cardano-ledger-shelley-ma-test >= 0.1.1.1
+  , cardano-ledger-shelley >= 0.1.1.1
+  , cardano-ledger-shelley-test >= 0.1.1.1
+  , cardano-data >= 0.1.1.1
+  , cardano-ledger-core >= 0.1.1.1
+  , cardano-ledger-pretty >= 0.1.1.1
+  , cardano-protocol-tpraos >= 0.1.1.1
+  , non-integral >= 0.1.1.1
+  , set-algebra >= 0.1.1.1
+  , small-steps >= 0.1.1.1
+  , small-steps-test >= 0.1.1.1
+  , vector-map >= 0.1.1.1
```

Comment:
@dnadales you should distribute these constraints in `ouroboros-consensus` packges.

@angerman I'd prefer `^>=` so we don't get CI build failures as soon as a new version of `cardano-ledger` or `plutus` is published to `CHaP.